src/js/components/page.js

```diff
@@ -2,6 +2,8 @@
 import React, { Component } from 'react';
 /*eslint-enable no-unused-vars*/
 
+import '../../css/page.css';
```

Comment:
I added browserify-css. Javascript modules import css files, and the browserify module loader injects the css into a `<style>` block. 
Now, we don't have to serve up individual css files, which cuts down on load time.